### PR TITLE
add trivy operator template for jira

### PR DESCRIPTION
**Pull request by martijnvdp**
copied the existing jira template and made it suitable for the trivy operator
and fixed the colomns and added summary with totals 
https://github.com/aquasecurity/postee/issues/598

### CODE REVIEW

File: rego-templates/trivy-operator-jira.rego [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+# builds 2-dimension array for vulnerability table
+vln_list(severity) = vlnrb {
+	some i
+	vlnrb := [r |
+		item := input.report.vulnerabilities[i]
+		vlnname := item.vulnerabilityID
+		title := item.title
+		fxvrsn := with_default(item, "fixedVersion", "none")
+		resource_name = with_default(item, "packageType", "none")
```
Comment (DmitriyLewen) [inline rego-templates/trivy-operator-jira.rego:116]: `packageType` is optional field - https://github.com/aquasecurity/trivy-operator/blob/c944cbfecfbe166f204e9f23e8a5f45d73cde787/pkg/apis/aquasecurity/v1alpha1/vulnerability_types.go#L93

Perhaps it is better to use only `resource` column with value from `resource`?

**Review by DmitriyLewen — COMMENTED** [review]
Hello @martijnvdp 
Left comments.

File: rego-templates/trivy-operator-jira.rego [hunk 2]
```diff
@@ -0,0 +1,140 @@
+package postee.trivyoperator.jira
+############################################# Common functions ############################################
+with_default(obj, prop, default_value) = default_value{
+not obj[prop]
+}
+with_default(obj, prop, default_value) = obj[prop]{
+obj[prop]
+}
```
[the remark this answers is not in the document]
Comment (martijnvdp) [reply rego-templates/trivy-operator-jira.rego]: changed it left it in for testing on rego playground

File: rego-templates/trivy-operator-jira.rego [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+to_severity_color(color, level) = spn {
+	spn := to_colored_text(color, format_int(with_default(input.report.summary,level, 0), 10))
+}
+
+render_summary := sprintf(sum_tpl,[
+	to_severity_color("#c00000", "criticalCount"),
+	to_severity_color("#e0443d", "highCount"),
+	to_severity_color("#f79421", "mediumCount"),
+	to_severity_color("#e1c930", "lowCount")
+])
```
[the remark this answers is not in the document]
Comment (martijnvdp) [reply rego-templates/trivy-operator-jira.rego]: added them nonecount

[the remark this answers is not in the document]
Comment (DmitriyLewen) [reply rego-templates/trivy-operator-jira.rego]: you need to add `unknownCount` - https://github.com/aquasecurity/trivy-operator/blob/37d454e631a9b538a6b4dd44cb3d65d188cfddf6/pkg/apis/aquasecurity/v1alpha1/vulnerability_types.go#L28

File: rego-templates/trivy-operator-jira.rego (hunk 1 above)
Comment (martijnvdp) [reply rego-templates/trivy-operator-jira.rego:118]: ah thanks i dindt checked the struct , took a example report as test input

**Review by DmitriyLewen — APPROVED** [review]
@martijnvdp Thanks for your work!
LGTM!
@simar7 can you take a look and merge this PR?

**Review by simar7 — APPROVED** [review]

**Pull request merged**